### PR TITLE
Updates Flash Messaging Bus doc for react-hot-toast

### PR DESCRIPTION
**Pull request by cannikin**
To be merged once https://github.com/redwoodjs/redwood/pull/1856 is released

### CODE REVIEW

File: docs/toastNotifications.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+Did you know that those little popup notifications that you sometimes see at the top of pages after you've performed an action are affectionately known as "toast" notifications? Because they pop up like a piece of toast from a toaster!
+
+![Example Toast Animation](https://user-images.githubusercontent.com/300/110032806-71024680-7ced-11eb-8d69-7f462929815e.gif)
+
+Redwood supports these notfications out of the box thanks to the [react-hot-toast](https://react-hot-toast.com/) package.
+
+## Usage
+
+This doc will not cover everything you can do with toasts, and the [react-hot-toast docs](https://react-hot-toast.com/docs) are very throurgh. But here are a couple of common use cases.
```
Comment (jtoar) [inline docs/toastNotifications.md:11]: - typo
```suggestion
This doc will not cover everything you can do with toasts, and the [react-hot-toast docs](https://react-hot-toast.com/docs) are very thorough. But here are a couple of common use cases.
```

**Comment by cannikin** [comment]
Thanks @jtoar!

File: docs/toastNotifications.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+## Usage
+
+This doc will not cover everything you can do with toasts, and the [react-hot-toast docs](https://react-hot-toast.com/docs) are very thorough. But here are a couple of common use cases.
+
+### Displaying a Toast
+
+Wherever you want your notifications to be output, include the **&lt;Toaster&gt;** component:
+
+```javsacript
+import { Toaster } from '@redwoodjs/web'
```
Comment (peterp) [inline docs/toastNotifications.md:18]:
```suggestion
import { Toaster } from '@redwoodjs/web/toast'
```

File: docs/toastNotifications.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+```
+
+See the [official Toaster docs](https://react-hot-toast.com/docs/toaster) for more options. There's also a [dedicated doc for styling](https://react-hot-toast.com/docs/styling).
+
+### Triggering a Toast
+
+To show a toast message, just include a call to the `toast` object:
+
+```javascript
+import { toast } from '@redwoodjs/web'
```
Comment (peterp) [inline docs/toastNotifications.md:57]:
```suggestion
import { toast } from '@redwoodjs/web/toast'
```

**Comment by cannikin** [comment]
Did we merge @Tobbe's suggestion of just exporting `*` from `react-hot-toast`? In that case we need to update these import statements to list `toast` as the default:

    import toast, { Toaster } from '@redwoodjs/web/toast'

**Comment by thedavidprice** [comment]
@cannikin I merged this since the release is out. I don't have an answer about exporting all the things ¯\_(ツ)_/¯

**Comment by Tobbe** [comment]
@thedavidprice @cannikin 

TL;DR: The doc is correct 👍 

We have this

```
export { default as toast } from 'react-hot-toast'
export * from 'react-hot-toast'
```

So the import should be like this, which is what we have here
```
import { toast, Toaster } from '@redwoodjs/web/toast'
```
